spec/index.bs

```diff
@@ -9112,6 +9366,24 @@ partial interface GPUDevice {
         - There are no error scopes on the stack.
 </dl>
 
+<div class="example">
+    Using error scopes to capture errors from a {{GPUDevice}} operation that may fail:
+    <pre highlight="js">
+        gpuDevice.pushErrorScope('validation');
+
+        let shadermodule = gpuDevice.createShaderModule({
```

Comment:
It's not super clear yet whether this is the "right" way to capture errors from shader modules. It might change - and also we may stop returning shader errors in error.message. Maybe pick a more evergreen example, like, createSampler with maxAnisotropy=0?